crates/composer/src/main.rs

```diff
@@ -0,0 +1,42 @@
+use composer::api::Event;
+use composer::api::DEFAULT_SERVER_ADDRESS;
+use eyre::Result;
+use rodio::{source::Source, Decoder, OutputStream, OutputStreamHandle};
+use std::fs::File;
+use std::io::BufReader;
+use std::net::UdpSocket;
+
+fn main() -> Result<()> {
+    color_eyre::install()?;
+
+    let socket = UdpSocket::bind(DEFAULT_SERVER_ADDRESS)?;
+    println!("Listening on {}", socket.local_addr()?);
+
+    let (_stream, stream_handle) = OutputStream::try_default()?;
+
+    loop {
+        if let Err(err) = handle_datagram(&socket, &stream_handle) {
+            eprintln!(
+                "Could not process datagram. Ignoring and continuing. {:?}",
+                err
+            );
+            continue;
```

Comment:
Minor nit: the `continue` isn't needed anymore.

Reply:
removed in [4b30780](https://github.com/tonarino/acoustic_profiler/pull/12/commits/4b307804325f81f77cf512369670df9a935824d6)